editor/src/clj/editor/code/script.clj

```diff
@@ -192,18 +192,25 @@
 (def resource-kind->ext
   "Declares which file extensions are valid for different kinds of resource
   properties. This affects the Property Editor, but is also used for validation."
-  {"atlas"       ["atlas" "tilesource"]
+  {"atlas"       "" ; dummy in order to make ResourceKind work
    "font"        "font"
    "material"    "material"
    "buffer"      "buffer"
    "texture"     (conj image/exts "cubemap" "render_target")
    "tile_source" "tilesource"})
```

Comment:
I think ideally, this file should keep the resource kinds that it supports, and always ask the workspace for the actual file suffixes.

But I opted to not go the full way there, as there might be other considerations to take.